src/inmanta/moduletool.py

```diff
@@ -603,14 +625,28 @@ def modules_parser_config(cls, parser: ArgumentParser) -> None:
         install.add_argument("-e", "--editable", action="store_true", help="Install in editable mode.")
         install.add_argument("path", nargs="?", help="The path to the module.")
 
-        subparser.add_parser("status", help="Run a git status on all modules and report")
+        subparser.add_parser(
+            "status",
+            help="Run a git status on all modules and report",
+            parents=[verbosity_parser],
+        )
 
-        subparser.add_parser("push", help="Run a git push on all modules and report")
+        subparser.add_parser(
+            "push",
+            help="Run a git push on all modules and report",
+            parents=[verbosity_parser],
+        )
 
         # not currently working
-        subparser.add_parser("verify", help="Verify dependencies and frozen module versions")
+        subparser.add_parser(
+            "verify",
+            help="Verify dependencies and frozen module versions",
+            parents=[verbosity_parser],
+        )
 
-        commit = subparser.add_parser("commit", help="Commit all changes in the current module.")
+        commit = subparser.add_parser(
+            "commit", help="Commit all changes in the current module.", parents=[long_verbosity_parser]
```

Comment:
A specific verbosity parser that only parses "--verbose" (and not "-v") is required for this sub-command since it already defines a "-v" option `commit.add_argument("-v", "--version", help="Version to use on tag")`

Reply:
As discussed on slack: leave commit command untouched, it is deprecated and will be dropped from iso7 in the near future.